docs/guides/deployment.md

```diff
@@ -33,21 +28,42 @@ helm install --namespace vvp \
             --set rbac.additionalNamespaces={top-speed}
 
 ## Or, for the Community Edition
+For helm 2:
+
+```bash
 helm install --namespace vvp \
             --name vvp \
             ververica/ververica-platform \
             --set acceptCommunityEditionLicense=true \
             --set rbac.additionalNamespaces={top-speed}
+```
 
-# 2. 
-# Wait for the deployment to come up
+For helm 3:
+
+```bash
+helm install vvp ververica/ververica-platform -n vvp --set acceptCommunityEditionLicense=true
+```
+
+## 2.Wait for the deployment to come up
+
+```bash
 kubectl --namespace vvp wait --for=condition=available deployments --all
+```
+
+## 3. Access the platform UI locally at port 8080
 
-# 3. 
-# Access the platform UI locally at port 8080
-kubectl port-forward --namespace vvp service/vvp-ververica-platform 8080:80 &
+```bash
+kubectl port-forward --namespace vvp service/vvp-ververica-platform 8080:80
 ```
 
+NOTE: If you don't have cert-manager you may get following error:
+
+```bash
+Error: Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=30s: no endpoints available for service "cert-manager-webhook"
+```
+
+To deploy cert-manager[`check out this guide`](./cert-manager.md)
```

Comment:
What do you think about moving this to the top of the guide where Cert-Manager is mentioned?